app/src/main/play/release-notes/en-US/default.txt

```diff
@@ -0,0 +1,27 @@
+New
+    Fresh design - completely rebuilt.
+    Ability to add Reactions to messages
+    Swipe left to reply
+    Button to scroll to end of conversation
+    Drafts : keep your message in text field after leaving conversation
+    Navigate to original message from a reply.
```

Comment:
which is correct? having `.` or not having? it's not constant.

Reply:
I just copied and pasted what product sent me tbh, i will unify all and remove the .